src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -315,12 +315,14 @@ public static void makeGenotypeCall(final int ploidy,
             gb.alleles(noCallAlleles(ploidy));
         } else if (assignmentMethod == GenotypeAssignmentMethod.USE_PLS_TO_ASSIGN ||
                     assignmentMethod == GenotypeAssignmentMethod.PREFER_PLS) {
-            if ( genotypeLikelihoods == null || !isInformative(genotypeLikelihoods) ) {
+            if (genotypeLikelihoods == null || !isInformative(genotypeLikelihoods)) {
                 if (assignmentMethod == GenotypeAssignmentMethod.PREFER_PLS) {
                     if (originalGT == null) {
                         throw new IllegalArgumentException("original GT cannot be null if assignmentMethod is PREFER_PLS");
+                    } else if (originalGT.hasGQ() && originalGT.getGQ() > 0){
```

Comment:
i see. this makes sense and was certainly better than my hack attempt to fix this... so its now "prefer_PLs EXCEPT for nocalls" Can you add a note about this in `GenotypeAssignmentMethod`?

Reply:
Is there a reason to preserve the intermediate behavior at all? We could split PREFER_PLS to also have a mode that treats the 0/0s like they used to be.